src/main/java/br/com/caelum/clines/api/locations/LocationViewMapper.java

```diff
@@ -1,4 +1,4 @@
-package br.com.caelum.clines.api.airports;
+package br.com.caelum.clines.api.locations;
```

Comment:
Same here, there is not used in the airport package anymore?

Reply:
(same answer above)

Reply:
@andrelbravo , fixed